src/System.Drawing.Common/src/mono/MonoTODOAttribute.cs

```diff
@@ -0,0 +1,111 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I'd be happy to remove them if mono isn't really using or tracking them. TBH, a lot of them just say "not implemented on Unix" when the implementation already throws a NotImplementedException. So many of them aren't really tracking anything meaningful anyways...

I will definitely make them conditional for us, at the very least.